src/background.js

```diff
@@ -42,7 +45,7 @@ async function spinWith(func, ...args) {
 }
 
 async function processSelectedMessages(folder, operation="move", goToFolder=false) {
-  let { markAsRead } = await browser.storage.local.get({ markAsRead: true });
+  let { markAsRead, notificationActive, op_counters } = await browser.storage.local.get({ markAsRead: true, notificationActive: DEFAULT_PREFERENCES.notificationActive, op_counters: DEFAULT_PREFERENCES.op_counters });
```

Comment:
Let's use `operationCounters` to make more clear what this is and consistency on case naming.

Reply:
ok, done.